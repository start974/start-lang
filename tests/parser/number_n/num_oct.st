def a := 0o52
def b := 0O0000_0052
