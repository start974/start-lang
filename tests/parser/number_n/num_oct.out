def a := 42
def b := 42
